label_studio_ml/examples/segment_anything_model/Dockerfile

```diff
@@ -3,31 +3,15 @@ FROM python:3.8-slim
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=True \
-    PORT=9090 \
-    VITH_CHECKPOINT=/app/sam_vit_h_4b8939.pth \
-    ONNX_CHECKPOINT=/app/sam_onnx_quantized_example.onnx \
-    MOBILESAM_CHECKPOINT=/app/mobile_sam.pt
+    PORT=9090
 
 COPY requirements.txt .
 
-RUN apt-get update && apt-get install -y wget git \
-    && pip install --no-cache-dir -r requirements.txt \
-    && wget https://dl.fbaipublicfiles.com/segment_anything/sam_vit_h_4b8939.pth \
-    && wget https://cdn.githubraw.com/ChaoningZhang/MobileSAM/01ea8d0f/weights/mobile_sam.pt
+RUN apt-get update && apt-get install -y wget git python3-opencv \
+    && pip install --no-cache-dir -r requirements.txt
 
-COPY onnxconverter.py .
-RUN python onnxconverter.py .
+COPY . ./
 
```

Comment:
I think we should download the model weights here using download.py, just to make the Docker file as useful as possible. I understand that this makes the Dockerfile much larger, but for beginning users who are just getting started, I believe that this is a better experience. More expert users will be able to modify the model to meet their particular needs.

Reply:
added `./download_models.sh` step in README